sdk/src/test/java/io/opentelemetry/sdk/trace/spi/TraceProviderSdkTest.java

```diff
@@ -14,18 +14,19 @@
  * limitations under the License.
  */
 
-package io.opentelemetry.sdk.trace;
+package io.opentelemetry.sdk.trace.spi;
 
 import static com.google.common.truth.Truth.assertThat;
 
 import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.sdk.trace.TracerSdkRegistry;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/** Unit tests for {@link SdkTraceProvider}. */
+/** Unit tests for {@link io.opentelemetry.sdk.trace.spi.TraceProviderSdk}. */
```

Comment:
not sure why this ended up fully-qualified. it shouldn't need to be.

Reply:
Done.